### ISSUE TITLE
Trying to remove liquidity using the "max" button sometimes gives an error

### ISSUE THREAD

**Issue by LucaMilanese90** [oceanprotocol/market#1164]
**Describe the bug**
When trying to remove liquidity from this asset [did:op:ca285f858304b20e4e319e9c1b17fcc60b8a2e08af8e52d88fd2b9427f9d5649](https://market-git-v4-oceanprotocol.vercel.app/asset/did:op:ca285f858304b20e4e319e9c1b17fcc60b8a2e08af8e52d88fd2b9427f9d5649) using the "max" button, the value is too high and an error is shown

**Screenshots**
![Screenshot 2022-03-03 at 16 48 59](https://user-images.githubusercontent.com/20192135/156616494-a15336a1-7c7d-4d8c-9fd5-b1ecbac7ec03.png)

**Comment by mihaisc** [comment]
When did you test this? Earlier this day a pr was merged that may have fixed this

**Comment by LucaMilanese90** [comment]
A few minutes ago, but maybe the preview link I used was not updated

**Comment by mihaisc** [comment]
ok, well investigate further.

**Comment by LucaMilanese90** [comment]
Just tried again on the same asset [did:op:ca285f858304b20e4e319e9c1b17fcc60b8a2e08af8e52d88fd2b9427f9d5649](https://market-git-v4-oceanprotocol.vercel.app/asset/did:op:ca285f858304b20e4e319e9c1b17fcc60b8a2e08af8e52d88fd2b9427f9d5649) and it's still happening.

**Comment by AnaLoznianu** [comment]
We need to double-check if this is still an issue

**Comment by LucaMilanese90** [comment]
Still happening on the asset from the previous comment and for others as well.
[did:op:99fd9981a1b6b0cdff2c88622ffd8a16c6f41952e8bcd62c786f97dc33fc0aee](https://market-git-v4-oceanprotocol.vercel.app/asset/did:op:99fd9981a1b6b0cdff2c88622ffd8a16c6f41952e8bcd62c786f97dc33fc0aee)
[did:op:5d17994417759e97f9429ee5605302371ace55e33f359a281e22c6f0d11cbdc1](https://market-git-v4-oceanprotocol.vercel.app/asset/did:op:5d17994417759e97f9429ee5605302371ace55e33f359a281e22c6f0d11cbdc1)

Also notice the weird movement of the max button on hover.

https://user-images.githubusercontent.com/20192135/157226525-c1a34de3-c409-4b00-911c-76871b30f36d.mov

**Comment by KatunaNorbert** [comment]
I can confirm the max button issue on hover, however the remove liquidity worked for me for each asset and haven't encountered your error.
An error that I encountered sometime is that the internal state is not updated with the new values after removing liquidity and I could access the remove page again as I haven't removed anything, and then when trying to remove, the transaction fails. I think we should wait for the state to get updated before showing that the liquidity removal has been successfully removed.

**Comment by kremalicious** [comment]
This might be also this interval problem where refetching updates state but then old interval kicks in. As far as I saw when I deactivated that interval things didn't go so weird. See https://github.com/oceanprotocol/market/issues/1152#issuecomment-1062854164

But also worth testing correct values after add/remove without the interval active to see if this described issue might be caused by that too. Cause we refetch specifically after add/remove liquidity already
